jsonschema-hyperschema.xml

```diff
@@ -116,12 +121,20 @@
                 The terms "schema" and "instance" are to be interpreted as defined in the
                 <xref target="json-schema">JSON Schema core specification</xref>.
             </t>
+
+            <t>
+                This specification uses the terms "link context" and "link target" as
+                defined in <xref target="I-D.nottingham-rfc5988bis"/>.  These terms respectively
+                correspond with "Context IRI" and "Target IRI" as used in
+                <xref target="RFC5988">RFC 5988</xref>.
+                Although defined as IRIs, in common scenarios they are also URIs.
```

Comment:
When it actually gets assigned a new RFC number, we will.  For now, the xref system needs a document it understands, and that's the I-D.  That's a "send after approval" form letter, not an actual indication of approval quite yet.